addons/web/static/src/views/calendar/calendar_model.js

```diff
@@ -209,6 +213,26 @@ export class CalendarModel extends Model {
         await this.orm.create(this.meta.resModel, [rawRecord], { context });
         await this.load();
     }
+
+    async multiCreateRecords(dates, values) {
+        const records = [];
+        const [section] = this.filterSections;
```

Comment:
```js
// we deliberately only use the values of the first filter section, to avoid combinatorial explosion
const [section] = this.filterSections;
```